apps/portal/app/lib/utils/misc.tsx

```diff
@@ -427,6 +428,106 @@ export const getAtomId = (atom: IdentityPresenter) => {
   return atom.identity_id
 }
 
+// atom GQL helpers
+export const getAtomImageGQL = (
```

Comment:
Amazing! I started adding some of these in the branch for Explore (my open PR). We can streamline once both are merged in and see if there are duplicates.